client/src/components/Applications/AppBrowser/AppBrowser.scss

```diff
@@ -29,13 +26,15 @@
   padding-bottom: var(--vertical-buffer);
 }
 
-#appBrowser-sidebar {
-    margin-left: -20px;
+#appBrowser-sidebar,
+#appBrowser-tray {
+  --horizontal-buffer: var(--global-space--section-left);
 }
-#appBrowser-sidebar .nav-item {
-  /* Define minimum width of list and minimum width of columns in list */
-  /* NOTE: Wes B. does not know how to do this using CSS Grid. Help is welcome. */
-  min-width: var(--min-width);
+#appBrowser-sidebar {
+  --min-width: 200px;
+
+  margin-left: calc(var(--horizontal-buffer) * -1);
+  border-right: 1px solid rgba(112, 112, 112, 0.25);
 }
```

Comment:
__Explanation__
Just trying to make app sidebar organize its styles more like other sidebars, even though it is unique.